README.rst

```diff
@@ -175,657 +175,1079 @@ automatically fix anything:
-                         space.
-
-* **blank_line_after_namespace** [@PSR2, @Symfony]
-                         There MUST be one blank line
-                         after the namespace
-                         declaration.
+                                        Binary
+                                        operators
+                                        should be
+                                        surrounded by
+                                        at least one
+                                        space.
+
+* **blank_line_after_namespace** [@PSR2,
+                                        @Symfony]
+                                        There MUST be
+                                        one blank line
+                                        after the
+                                        namespace
+                                        declaration.
 
 * **blank_line_after_opening_tag** [@Symfony]
-                         Ensure there is no code on
-                         the same line as the PHP open
-                         tag and it is followed by a
-                         blankline.
+                                        Ensure there
+                                        is no code on
+                                        the same line
+                                        as the PHP
+                                        open tag and
+                                        it is followed
+                                        by a
+                                        blankline.
 
 * **blank_line_before_return** [@Symfony]
-                         An empty line feed should
-                         precede a return statement.
-
-* **braces** [@PSR2, @Symfony]
-                         The body of each structure
-                         MUST be enclosed by braces.
-                         Braces should be properly
-                         placed. Body of braces should
-                         be properly indented.
+                                        An empty line
+                                        feed should
+                                        precede a
+                                        return
+                                        statement.
+
+* **braces** [@PSR2,
+                                        @Symfony]
+                                        The body of
+                                        each structure
+                                        MUST be
+                                        enclosed by
+                                        braces. Braces
+                                        should be
+                                        properly
+                                        placed. Body
+                                        of braces
+                                        should be
+                                        properly
+                                        indented.
```

Comment:
i dont like putting one word per statement...
sth to be resolved in new PR.

nothing to worry about for you @ro0NL ;)